src/main/java/com/mathworks/ci/actions/RunMatlabTestsAction.java

```diff
@@ -51,13 +50,7 @@ public void run() throws IOException, InterruptedException, MatlabExecutionExcep
                 .println(e.getMessage());
             throw(e);
         } finally {
-            try {
-                this.runner.removeTempFolder();
-            } catch (Exception e) {
-                // Don't want to override more important error
-                // thrown in catch block
-                System.err.println(e.toString());
-            }
+            super.teardownAction();
```

Comment:
```suggestion
            teardownAction();
```

Reply:
I agree `super.teardown()` is not mandatory but slightly helps the reader so would prefer to keep it